src/OSPSuite.Core/Domain/Services/ParameterIdentificationValidator.cs

```diff
@@ -58,15 +55,18 @@ private static IEnumerable<IdentificationParameter> allIdentificaitonParametersW
 
       private void validateWeights(ParameterIdentification parameterIdentification, ValidationResult validationResult)
       {
-         if (allWeightedDataRepositoryWeights(parameterIdentification).Where(weight => weight < 0).Union(allDataPointWeights(parameterIdentification).Where(weight => weight < 0)).Any())
+         var allDataRepositoryWeights = allWeightedDataRepositoryWeights(parameterIdentification);
+         var allPointWeights = allDataPointWeights(parameterIdentification);
+
+         if (allDataRepositoryWeights.Union(allPointWeights).Any(weight => weight < 0))
          {
             validationResult.AddMessage(NotificationType.Error, parameterIdentification, Error.WeightValueCannotBeNegative);
          }
       }
 
       private static IEnumerable<float> allDataPointWeights(ParameterIdentification parameterIdentification)
       {
-         return parameterIdentification.OutputMappings.All.SelectMany(x => x.WeightedObservedData.Weights);
+         return parameterIdentification.OutputMappings.All.Where(x => x.WeightedObservedData != null).SelectMany(x => x.WeightedObservedData.Weights);
```

Comment:
That guys could be null if not mapped. I added a test for that and fixed the issue